EventFilter/EcalRawToDigi/plugins/MatacqProducer.cc

```diff
@@ -546,7 +546,8 @@ bool MatacqProducer::getMatacqFile(uint32_t runNumber, uint32_t orbitId, bool* f
   //we make two iterations to handle the case where the event is procesed
   //before the matacq data are available. In such case we would have
   //orbitId > maxOrb (maxOrb: orbit of last written matacq event)
-  for (int itry = 0; itry < 2 && (orbitId > maxOrb); ++itry) {
+  //  for(int itry = 0; itry < 2 && (orbitId > maxOrb); ++itry){
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

the same applies to all other cases